ros_packages/motors/pib_motors/setup.py

```diff
@@ -1,8 +1,30 @@
-from setuptools import setup
+import os
+from glob import glob
+
+from setuptools import find_packages, setup
+
+package_name = "motors"
```

Comment:
why did the package-name change from "pib_motors" to "motors"?

Reply:
The line might have been highlighted as changed due to the new linter, but the package-name is called `motors` on `main` as well.
I think it happened during the migration to the monorepo, because the folder got renamed from `pib_motors` to `motors` as well.